javascript/javascript3/week1/WeatherApi/main.js

```diff
@@ -0,0 +1,87 @@
+const cityInput = document.getElementById('input-city')
+const btnSubmit = document.getElementById('btn-submit')
+const weatherDesUL = document.getElementById('weather-dec')
+const disTemp = document.querySelector('div .temp')
+const disCityDetail = document.querySelector('div .city-detail')
+const bgImg = document.querySelector('div .dis-part1')
+const disIcon = document.querySelector('div .weather-Icon')
+const disSunTime = document.getElementById('sun-time')
+const gmtDateTime = new Date().toUTCString();
+const apiKey = `aff81a037c13a860266157fd7cec79ed`
+let inputCityName = "copenhagen"
+btnSubmit.addEventListener('click', () => {
+    inputCityName= cityInput.value;
+    console.log(inputCityName)
+    if (!inputCityName) {
+        alert('enter city name')
+    }
+     display();
+});
+const display = () =>{
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${inputCityName}&units=metric&appid=${apiKey}`)
+    .then(response => response.json())
+    .then(weatherData => {
+        const data = JSON.stringify(weatherData)
```

Comment:
the data variable is never used

Reply:
yah, I have written for testing purpose in console. I have removed from code.